prisma/schema.prisma

```diff
@@ -6,6 +6,10 @@ generator trpc {
   provider          = "node ./lib/generator.js"
   isGenerateSelect  = true
   isGenerateInclude = true
+  withMiddleware = false
+  withShield     = false
+  contextPath    = "../../../../example/context"
+  v10           = true
```

Comment:
No need for the flag anymore. So all changes related below should happen in the original functions(without the v10 duplication)